djstripe/models.py

```diff
@@ -796,19 +827,20 @@ def is_valid(self):
         return True
 
     def update(self, prorate=djstripe_settings.PRORATION_POLICY, **kwargs):
-        # Convert Plan to stripe_id
+        """Convert Plan to stripe_id."""
         if "plan" in kwargs and isinstance(kwargs["plan"], Plan):
             kwargs.update({"plan": kwargs["plan"].stripe_id})
 
         stripe_subscription = super(Subscription, self).update(prorate=prorate, **kwargs)
         return Subscription.sync_from_stripe_data(stripe_subscription)
 
     def extend(self, delta):
+        """Call extend from Subscription and sync with Stripe api."""
         stripe_subscription = super(Subscription, self).extend(delta)
         return Subscription.sync_from_stripe_data(stripe_subscription)
 
     def cancel(self, at_period_end=djstripe_settings.CANCELLATION_AT_PERIOD_END):
-        # If plan has trial days and customer cancels before trial period ends, then end subscription now, i.e. at_period_end=False
+        """If plan has trial days and customer cancels before trial period ends, then end subscription now, i.e. at_period_end=False."""
```

Comment:
inherited; revert (as above, the comments are for the conditional, not the method)